README.md

```diff
@@ -1,11 +1,11 @@
 # Mry
 
-Mry Migrates .Rubocop.Yml :muscle:
+Mry Migrates `.rubocop.yml` :muscle:
```

Comment:
Could you revert this change? The capital characters mean acronym.

Migrates
Rubocop
Yml